component/dependencies/cloudscale-metrics-collector/component/main.jsonnet

```diff
@@ -0,0 +1,108 @@
+local kap = import 'lib/kapitan.libjsonnet';
+local inv = kap.inventory();
+local params = inv.parameters.cloudscale_metrics_collector;
+
+local namespace = 'appuio-cloud-reporting';
+local image = 'ghcr.io/vshn/cloudscale-metrics-collector:v0.2.0';
```

Comment:
Syn expects that images and charts are defined in a certain way and configurable, so that Renovate can properly pick up and create PRs in case there is a newer image.
See https://syn.tools/syn/explanations/commodore-components/container-images.html

Reply:
Done.